lib/telemetry_metrics.ex

```diff
@@ -124,17 +125,29 @@ defmodule Telemetry.Metrics do
       summary("http.request.stop.duration", unit: {from_unit, to_unit})
 
   This means that the measurement will be converted from `from_unit` to `to_unit` before
-  being used for updating the metric. Currently, only time conversions are supported,
-  which means that both `from_unit` and `to_unit` need to be one of `:second`, `:millisecond`,
-  `:microsecond`, `:nanosecond`, or `:native`. That's because most time measurements
-  in the Erlang VM are done in the `:native` unit, which we need to convert to the desired
-  precision.
+  being used for updating the metric. Currently, only time and storage conversions are
+  supported.
+
+  ### Time Conversions
+
+  Most time measurements in the Erlang VM are done in the `:native` unit, which we need
+  to convert to the desired precision. The supported time units are: `:second`, `:millisecond`,
+  `:microsecond`, `:nanosecond` and `:native`.
 
   For example, to convert HTTP request duration from `:native` time unit to milliseconds
   you'd write:
 
       summary("http.request.stop.duration", unit: {:native, :millisecond})
 
+  ### Storage Conversions
```

Comment:
I wasn't sure which noun to use to call this group of units. I decided to go with `storage` but I'm not very happy with it. Maybe someone has a better suggestion?

Reply:
Byte units?